gateway/src/config/old_config_v1_1_29.rs

```diff
@@ -164,6 +164,10 @@ impl From<ConfigV1_1_29> for Config {
                 stored_messages_filename_length: value.debug.stored_messages_filename_length,
                 message_retrieval_limit: value.debug.message_retrieval_limit,
                 use_legacy_framed_packet_version: value.debug.use_legacy_framed_packet_version,
+
+                // \/ ADDED
+                // enforce forward travel, et al.
+                ..Default::default() // /\ ADDED
```

Comment:
hmm ... can we get away with this? is all that conversion code in my PR not needed?